pkg/tests/apis/secret/keeper_test.go

```diff
@@ -222,4 +222,122 @@ func TestIntegrationKeeper(t *testing.T) {
 			require.Empty(t, rawList.Items)
 		})
 	})
+
+	t.Run("creating keepers in multiple namespaces", func(t *testing.T) {
+		adminOrg1 := helper.Org1.Admin
+		adminOrgB := helper.OrgB.Admin
+
+		keeperOrg1 := mustGenerateKeeper(t, helper, adminOrg1, nil)
+		keeperOrgB := mustGenerateKeeper(t, helper, adminOrgB, nil)
+
+		clientOrg1 := helper.GetResourceClient(apis.ResourceClientArgs{User: adminOrg1, GVR: gvrKeepers})
+		clientOrgB := helper.GetResourceClient(apis.ResourceClientArgs{User: adminOrgB, GVR: gvrKeepers})
+
+		// Create
+		t.Run("creating a keeper with the same name as one from another namespace does not return an error", func(t *testing.T) {
+			// Org1 creating a keeper with the same name from OrgB.
+			testData := helper.LoadYAMLOrJSONFile("testdata/keeper-aws-generate.yaml")
+			testData.SetName(keeperOrgB.GetName())
+
+			raw, err := clientOrg1.Resource.Create(ctx, testData, metav1.CreateOptions{})
+			require.NoError(t, err)
+			require.NotNil(t, raw)
+
+			require.NoError(t, clientOrg1.Resource.Delete(ctx, raw.GetName(), metav1.DeleteOptions{}))
```

Comment:
Wouldn't it be better to delete only at the end to ensure both keepers exist at the same time?

Reply:
Done!